modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/TcsEpics.scala

```diff
@@ -281,6 +281,9 @@ final class TcsEpics(epicsService: CaService) {
 
   def inPosition: Option[String] = Option(inPositionAttr.value)
 
+  val agInPositionAttr: CaAttribute[java.lang.Double] = tcsState.getDoubleAttribute("agInPosition")
```

Comment:
I agree it should be private, but not in a block, because it is not only used in `agInPosition`